lib/internal/test_runner/tests_stream.js

```diff
@@ -29,6 +29,10 @@ class TestsStream extends Readable {
     }
   }
 
+  lifecycleRun(nesting, file, testNumber, name, details) {
+    this[kEmitMessage]('test:lifecycle:run', { __proto__: null, name, nesting, file, testNumber, details });
```

Comment:
I think `test:enqueue` is a better name, @cjihrig WDYT?

Reply:
I don't feel too strongly, but I wouldn't say "enqueue" just rolls off the tongue.

Reply:
I agree, but in terms of distinction from `test:start` I think it is much better than `test:lifecycle:run`